app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -64,5 +65,17 @@ def stop_event_monitor_queue_on_credential_change
     def supports_cloud_tenants?
       false
     end
+
+    def sync_cloud_tenants_with_tenants
+      return unless supports_cloud_tenants?
```

Comment:
maybe you want to refactor `supports_cloud_tenants?` to use the [SupportsFeatureMixin](https://github.com/ManageIQ/manageiq/pull/8886) ? 
Or do this in a different PR. It would also be great to see in the `supports` block, what it actually takes to support cloud tenancy :smile: